addons/mail/static/src/core/common/message_model.js

```diff
@@ -383,6 +383,12 @@ export class Message extends Record {
             if (this.notificationType === "call") {
                 return _t("%(caller)s started a call", { caller: this.authorName });
             }
+            if (this.notificationType === "thread_deletion") {
+                return _t('%(user)s deleted the thread "%(thread_name)s"', {
+                    user: this.authorName,
+                    thread_name: this.body,
```

Comment:
`thread_name` should be this.thread.(display)name instead?

It needs to be dynamic, and also this.body might contain many other things no? Even the html itself?

Reply:
Ok I get it, it's the deleted thread name, not the current thread.
But I still think body will contain html?

Reply:
I saw that too, but I don't get why it's a "body"? It looks like a bad name somehow?

Reply:
`body` is not the prettiest, I could retrieve the notification text content in a similar way as `notificationType` if you think this is better. Putting it as attribute has the drawback that we have to use an attribute that passes the html sanitizer and current allowed-list has no great candidate for something like `data-o-thread-name`.

Reply:
It's fine to use body for now, but it needs to take the text content of the body (or `decorateEmojis(htmlToTextContentInline` as done in the last return)? And not the body itself.
This currently includes all original HTML which we specifically do not want for "inline" body.